test/README.md

```diff
@@ -1,17 +1,19 @@
 ## How to execute tests
 
+It requires [vim-themis](https://github.com/thinca/vim-themis).
+
 ```console
-$ bundle install --path=.bundle
-$ rake test
+$ git clone https://github.com/thinca/vim-themis
+$ ./vim-themis/bin/themis .
```

Comment:
Please add `$ cd /path/to/clever-f.vim` line before `git clone` to clarify where the current working directory is.